### DISCUSSION TITLE
Creation of Equivalent Fork/Shifting Base in Future, if Red Hat like situation occur

### DISCUSSION THREAD

**Discussion by AssassinK786** [linuxmint/discussions#52] [Questions] [answered]
https://www.phoronix.com/news/Red-Hat-CentOS-Stream-Sources

For now, Ubuntu may not do this type of work, but seeing the Ubuntu's Wired Decision Taking and possibility that it may be happen, it will better to create equivalent fork/shift to different base i.e Debian/create own base so that it will not affect the work progress

**Comment by rcalixte** [comment, marked as answer]
This is what LMDE is.

[LMDE - Wikipedia](https://en.wikipedia.org/wiki/Linux_Mint#LMDE)
[LMDE - Downloads](https://linuxmint.com/download_lmde.php)
